OpenAgency.md

```diff
@@ -17,18 +17,27 @@ requests can be found below.
 
 ## Versions
 
+OpenAgency 3.1 is considered the primary production version.
+
+<span style="color:red">End-of-life date for versions older than 3.1 is October 1st, 2021.</span>
+
+## Versions
+
 | Version | Environment | Endpoint | WSDL | XSD |
 |---------|-------------|----------|------|-----|
+| 3.1    | production  | [http://openagency.addi.dk/3.1/](http://openagency.addi.dk/3.1/) | [http://openagency.addi.dk/3.1/soap?wsdl](http://openagency.addi.dk/3.1/soap?wsdl) | [http://openagency.addi.dk/3.1/soap?xsd=1](http://openagency.addi.dk/3.1/soap?xsd=1) |
+| 3.1    | extern test | [http://openagency.test.addi.dk/3.1/](http://openagency.test.addi.dk/3.1/) | [http://openagency.test.addi.dk/3.1/soap?wsdl](http://openagency.test.addi.dk/3.1/soap?wsdl) | [http://openagency.test.addi.dk/3.1/soap?xsd=1](http://openagency.test.addi.dk/3.1/soap?xsd=1) |
+| 3.1    | staging     | [http://openagency.stg.addi.dk/3.1/](http://openagency.stg.addi.dk/3.1/) | [http://openagency.stg.addi.dk/3.1/soap?wsdl](http://openagency.stg.addi.dk/3.1/soap?wsdl) | [http://openagency.stg.addi.dk/3.1/soap?xsd=1](http://openagency.stg.addi.dk/3.1/soap?xsd=1) |
 | 3.0    | production  | [http://openagency.addi.dk/3.0/](http://openagency.addi.dk/3.0/) | [http://openagency.addi.dk/3.0/soap?wsdl](http://openagency.addi.dk/3.0/soap?wsdl) | [http://openagency.addi.dk/3.0/soap?xsd=1](http://openagency.addi.dk/3.0/soap?xsd=1) |
 | 3.0    | extern test | [http://openagency.test.addi.dk/3.0/](http://openagency.test.addi.dk/3.0/) | [http://openagency.test.addi.dk/3.0/soap?wsdl](http://openagency.test.addi.dk/3.0/soap?wsdl) | [http://openagency.test.addi.dk/3.0/soap?xsd=1](http://openagency.test.addi.dk/3.0/soap?xsd=1) |
 | 3.0    | staging     | [http://openagency.stg.addi.dk/3.0/](http://openagency.stg.addi.dk/3.0/) | [http://openagency.stg.addi.dk/3.0/soap?wsdl](http://openagency.stg.addi.dk/3.0/soap?wsdl) | [http://openagency.stg.addi.dk/3.0/soap?xsd=1](http://openagency.stg.addi.dk/3.0/soap?xsd=1) |
 
 ### 3.1 info
-OpenAgency 3.1 is currently in development and will be available soon.
+OpenAgency 3.1 is available as of 10/06/2021 (10th of June 2021).
 
-The following change is planned for version 3.1:
+The following change is new for version 3.1:
 * The pickupAgency element of the findLibrary, getRegistryInfo and pickupAgencyList
-methods are expanded with a routeNumber element.
+methods is expanded with a routeNumber element.
```

Comment:
nitpick, is er til ental, are er til flertal, så hvis det er method skal det være are.

Reply:
Det var også det jeg først skrevet men det "is" henviser til "The pickupAgency element" som er ental.
Jeg refererer stadig til de tre metoder som flertal "the findLibrary, getRegistryInfo and pickupAgencyList methods".
Så jeg mener stadig is er det korrekte.